### PR TITLE
Define file system operation interface

### PR DESCRIPTION
**Pull request by HHoflittlefish777**
related issue https://github.com/apache/skywalking/issues/10839

### CODE REVIEW

File: dist/LICENSE [hunk 1]
```diff
@@ -456,6 +456,7 @@ MIT licenses
     fill-range 7.0.1 MIT
     follow-redirects 1.15.2 MIT
     form-data 4.0.0 MIT
+    fsevents 2.3.2 MIT
```
Comment (hanahmily) [inline dist/LICENSE:459]: Where is it from? You've not introduced it in `go.mod` yet.

**Review by hanahmily — CHANGES_REQUESTED** [review]
The interface is good for me. But I have no idea where is the added `fsevents` from.

File: dist/LICENSE (hunk 1 above)
Comment (HHoflittlefish777) [reply dist/LICENSE:459]: This is I run make license-dep and it generate automatically

Comment (hanahmily) [reply dist/LICENSE:459]: Please remove your local `<banyandb_root_dir>/bin`. And rerun this command.

Comment (HHoflittlefish777) [reply dist/LICENSE:459]: The result of removing and rerunning make license-dep is the same.

**Review by hanahmily — APPROVED** [review]

**Pull request merged**